### PR TITLE
LPS-149551: Testray Manage Users User Self Information

### PR DESCRIPTION
**Pull request by caiosouza15**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Manage/User/UserForm.tsx [hunk 1]
```diff
@@ -215,19 +194,17 @@ const UserForm = () => {
...
 								<ClayCheckbox
-									checked={rolesWatch.includes(id)}
+									checked={userRoles.includes(id)}
+									disabled={!userRoles.includes(id)}
 									label={name}
-									onChange={(event) =>
-										setCheckedValue(event.target.value)
-									}
-									value={id}
+									onChange={() => {}}
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Manage/User/UserForm.tsx:200]: Remove this line

```suggestion
									
```

File: modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Manage/User/UserForm.tsx [hunk 2]
```diff
@@ -215,19 +194,17 @@ const UserForm = () => {
...
 								/>
 							</div>
 						))}
 					</ClayLayout.Col>
 				</ClayLayout.Row>
 
-				<Form.Divider />
+				<br />
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Manage/User/UserForm.tsx:207]: No need for this, maybe should be better to put a margin bottom in <ClayLayout.Row className="mb-2"

```suggestion
			
```

File: modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/schema/yup.ts [hunk 3]
```diff
@@ -20,10 +20,15 @@ import i18n from '../i18n';
...
-	givenName: yup.string().required(),
+	alternateName: yup
+		.string()
+		.required(i18n.translate('this-field-is-required')),
+	emailAddress: yup
+		.string()
+		.email()
+		.required(i18n.translate('this-field-is-required')),
+	familyName: yup.string().required(i18n.translate('this-field-is-required')),
+	givenName: yup.string().required(i18n.translate('this-field-is-required')),
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/schema/yup.ts:31]: Create a Constant for this Required Text, before line 20

```const DEFAULT_REQUIRED_TEXT = i18n.translate('this-field-is-required');```

Put the same **CONSTANT** in other places that may use ```i18n.translate('this-field-is-required');``` 

```suggestion
	alternateName: yup
		.string()
		.required(DEFAULT_REQUIRED_TEXT),
	emailAddress: yup
		.string()
		.email()
		.required(DEFAULT_REQUIRED_TEXT),
	familyName: yup.string().required(DEFAULT_REQUIRED_TEXT),
	givenName: yup.string().required(DEFAULT_REQUIRED_TEXT),
```

File: modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Manage/User/UserForm.tsx [hunk 4]
```diff
@@ -27,7 +27,7 @@ import yupSchema, {yupResolver} from '../../../schema/yup';
 import {liferayUserAccountsRest} from '../../../services/rest';
 import {RoleTypes} from '../../../util/constants';
 
-type UserFormDefault = {
+export type UserFormDefault = {
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Manage/User/UserForm.tsx:30]: This Type `UserFormDefault` is used somewhere? if not, we should remove

**Review by kevenleone — CHANGES_REQUESTED** [review]
@caiosouza15 take a look

**Comment by kevenleone** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**